cardano-db-sync/test/Cardano/DbSync/Gen.hs

```diff
@@ -103,6 +104,7 @@ syncNodeConfig loggingCfg =
     <*> triggerHardFork
     <*> triggerHardFork
     <*> syncInsertOptions
+    <*> pure []
```

Comment:
It would probably be useful to generate `ipfs_gateway` values

Reply:
Any suggestions for a generator to be used here? Not bvery familiar with these tests